stable/weave-scope/charts/weave-scope-frontend/templates/deployment.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    {{- include "weave-scope.helm_std_labels" . | indent 4 }}
+    component: frontend
+  name: {{ template "weave-scope.fullname" . }}-app
+  annotations:
+    {{- include "weave-scope.annotations" . | indent 4 }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ template "weave-scope.name" . }}
+      release: {{ .Release.Name }}
+      component: frontend
+  template:
+    metadata:
+      labels:
+        {{- include "weave-scope.helm_std_labels" . | indent 8 }}
+        component: frontend
+    spec:
+      containers:
+        - name: app
+          image: {{ required "You must specify the desired container image" .Values.global.image }}:{{ required "You must specify the desired container image tag" .Values.global.imageTag }}
```

Comment:
Image and tag are already specified in `values.yaml`. I would remove the `required` calls.